docs/getting-started/server/guide.md

```diff
@@ -272,9 +270,15 @@ Navigate to the `dev` folder in your server repo and perform the following comma
 
 ## Install Licensing Certificate
 
-To run your local server environment as a licensed instance, you will need to download the
-`Licensing Certificate - Dev` from the shared Engineering collection and install it. This can be
-done by double-clicking on the downloaded certificate.
+To run your local server environment as a licensed instance:
+
+1. Go to Bitwarden's shared Development collection. (If you don't have access to this collection,
+   contact our IT manager. Make sure you have first set up a Bitwarden account using your company
+   email address.)
+2. Search for the `Licensing Certificate - Dev` credentials.
+3. Download the `dev.pfx` certificate and double-click to install it. You will be prompted to enter
```

Comment:
I used the `dev.pfx` file, but not sure if the `dev.cer` file would work fine as well?

Reply:
I'm not sure. To be honest I don't even know what the licensing certificate does.